packages/rich-text/src/plugins/List/__tests__/createListPlugin.test.tsx

```diff
@@ -105,3 +105,228 @@ describe('normalization', () => {
     assertOutput({ input, expected });
   });
 });
+
+describe('insertBreak', () => {
```

Comment:
I really like this, lots of detailed case tested here without the overhead of Cypress :+1:

**Nit:** Wouldn't it more sense to make this more human-readable rather than referring to the function we use to simulate a paragraph? E.g.

```suggestion
describe('insert line break', () => {
```

Reply:
I usually use the function name in unit tests to easily find it later. No strong opinion here. Will change it